### PR TITLE
CR-1093062 Typo in dmesg command instructions

### PR DESCRIPTION
**Pull request by sadasivanshaiju**
This  is a typo fix. It has been fixed in 3 files.

### CODE REVIEW

**Comment by xdavidz** [comment]
@maxzhen could you please add @sadasivanshaiju into the github system so that he can request code reviews in the future?

**Comment by xdavidz** [comment]
> Can one of the admins verify this patch?

retest this please

**Comment by larry9523** [comment]
ok to test

**Review by maxzhen — APPROVED** [review]

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c [hunk 1]
```diff
@@ -4495,7 +4495,7 @@ static void clock_status_check(struct platform_device *pdev, bool *latched)
 		if (status & 0x1) {
```
Comment (xdavidz) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c:4495]: Not for this CR, but in the future we should avoid dup code in xmc.c which should be done in clock.c. My future refactoring work will count this in.

**Review by xdavidz — APPROVED** [review]

**Pull request merged**